### PR TITLE
consul-svc-healthy: use system jq if available

### PR DESCRIPTION
**Pull request by anonymouse64**
If we have a system jq, for example on some images jq is already installed, use that one instead of trying to use the one from the consul-scripts volume because that one will only work with Alpine/musl based images.
For example the docker-edgex-mongo image needs to use this script but has Ubuntu as a base image which is glibc based and thus incompatible with the jq we have staged in the consul-scripts volume. So for that image, we should instead use jq from the base image.

### CODE REVIEW

**Comment by bnevis-i** [comment]
This PR is blocking validation of https://github.com/edgexfoundry/developer-scripts/pull/175

cc: @tingyuz @tsconn23

**Review by bnevis-i — APPROVED** [review]
Would approve if had committer rights.

**Comment by tsconn23** [comment]
@tingyuz Needs review and approval

**Review by tingyuz — APPROVED** [review]

**Pull request merged**
